src/main.zig

```diff
@@ -439,6 +439,8 @@ const usage_build_generic =
     \\  -fno-unwind-tables        Never produce unwind table entries
     \\  -fLLVM                    Force using LLVM as the codegen backend
     \\  -fno-LLVM                 Prevent using LLVM as the codegen backend
+    \\  -flibLLVM                 Force using LLVM shared library apias the codegen backend
+    \\  -fno-libLLVM              Prevent using LLVM as the codegen backend
```

Comment:
`s/Prevent using LLVM as the codegen backend/Prevent using LLVM shared library as the codegen backend`.